src/processFile.ts

```diff
@@ -102,32 +98,34 @@ function adjustImportExportStatements(src: SourceFile, options: ProcessFileOptio
                 }
             }
             const newRef = calcRelativeImportFilename(reference, srcFilename, srcRoot, targetRoot, ext);
```

Comment:
```suggestion
            const newRef = calcRelativeImportFilename(reference, srcFilename, srcRoot, targetRoot, ext);
            magicString.update(start, end, newRef);
```

Add back the `magicString.update`